src/bin/lpython.cpp

```diff
@@ -1562,6 +1568,7 @@ int main(int argc, char *argv[])
         app.add_flag("--get-rtl-dir", print_rtl_dir, "Print the path to the runtime library file");
         app.add_flag("--verbose", compiler_options.verbose, "Print debugging statements");
         app.add_flag("--enable-cpython", compiler_options.enable_cpython, "Enable CPython runtime");
+        app.add_flag("--enable-symengine", compiler_options.enable_symengine, "Enable Symengine runtime");
```

Comment:
I get it now, we haven't to specify `--enable-symengine` option to integration_tests

Reply:
Similar to this: 
```console
$ lpython --backend=c integration_tests/symbolics_01.py       
symbolics_01__tmp__generated__.c:2:10: fatal error: 'symengine/cwrapper.h' file not found
#include <symengine/cwrapper.h>
...
```